### ISSUE TITLE
[FROZEN] Iteration Plan for Summer 2020

### ISSUE THREAD

**Issue by KexyBiscuit** [AOSC-Dev/aosc-os-abbs#2202]
This plan captures our work in **Summer**.

## Important dates

- July 8, 2020: Start the first survey for updates. 
- September 29, 2020: Freeze Testing-Proposed.
  - Merge all changes to Testing for updates.
  - Transfer all skip-ahead changes to Experimental.
  - All non-complete entries on Testing-Proposed => Testing to be deferred to Fall 2020.
- September 29, 2020: Commence draft on Fall 2020 Iteration Plan.
- October 13, 2020: Merge all changes to Stable, iteration completion.

### Stable-Proposed => Stable

TBD

### Testing-Proposed => Testing

TBD

## Priority Tasks

Tasks listed in the following sections will be subject to prioritized processing. These items must be complete before the cycle completes, and is therefore recommended that packagers work on these entries before others.

### Stable-Proposed => Stable

- [x] June 5, 2020: Pale Moon v28.10.0 [Pale Moon 28.10.0](https://github.com/MoonchildProductions/Pale-Moon/releases/tag/28.10.0_Release)
  - [x] `palemoon`
  - *Contains security updates.* (#2208)
- [x] June 22, 2020: Chromium, Google Chrome v83.0.4103.116 [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2020/06/stable-channel-update-for-desktop_22.html)
  - [x] `chromium`
  - [x] `google-chrome`
  - *Contains security updates.* (#2196)
- [x] June 23, 2020: V2Ray v4.25.1 [v4.25.1](https://github.com/v2ray/v2ray-core/releases/tag/v4.25.1)
  - [x] `v2ray`
  - *Contains security updates.* (#2195)
- [x] June 24, 2020: Vivaldi v3.1.1929.45 [Minor update (3) for Vivaldi Desktop Browser 3.1](https://vivaldi.com/zh-hans/blog/news/minor-update-3-for-vivaldi-desktop-browser-3-1/)
  - [x] `vivaldi`
  - *Contains security updates.* (#2196)

### Testing-Proposed => Testing

- [ ] April 24, 2020: LxQt v0.15.0 [0.15.0](https://github.com/lxqt/lxqt/releases/tag/0.15.0)
  - [ ] `groups/lxqt`
- [ ] 2020?: GNOME v3.36.2
  - [ ] `groups/gnome`
- [ ] X11 Libraries, Fonts, Applications components update.
  - [ ] `x11-lib+32`
  - [ ] `x11-font`

### Stable-Proposed => Stable

- [ ] August 25, 2019: Aptitude v0.8.12
  - [ ] `aptitude`
- [ ] September 5, 2019: ZeroTier One v1.4.6 [Version 1.4.6](https://github.com/zerotier/ZeroTierOne/releases/tag/1.4.6)
  - [ ] `zerotier-one`
- [ ] September 22, 2019: `icdiff` v1.9.5
- [ ] October 21, 2019: Asunder v2.9.5 [Change log](http://littlesvr.ca/asunder/releases/ChangeLog)
  - [ ] `asunder`
- [ ] October 24, 2019: VirtualGL v2.6.3 [2.6.3](https://github.com/VirtualGL/virtualgl/releases/tag/2.6.3)
  - [ ] `virtualgl`
  - [ ] `virtualgl+32`
- [ ] November 1, 2019: Texmaker v5.0.4 [Changelog](https://www.xm1math.net/texmaker/log.html)
  - [ ] `texmaker`
- [ ] November 7, 2019: Gnumeric v1.12.46 [Gnumeric 1.12.46 aka "TBD" is now available](http://www.gnumeric.org/announcements/1.12/gnumeric-1.12.46.html)
  - [ ] `gnumeric`
- [ ] November 10, 2019: `arp-scan` v1.9.7
- [ ] November 15, 2019: Fisher v3.2.10
  - [ ] `fisherman`
  - *Provide `fisher`, as it is now renamed.*
- [ ] November 22, 2019: `restic` v0.9.6 [restic 0.9.6](https://github.com/restic/restic/releases/tag/v0.9.6)
- [ ] November 27, 2019: Zopfli v1.0.3
  - [ ] `zopfli`
- [ ] November 28, 2019: Zypper v1.14.33
  - [ ] `zypper`
- [ ] December 4, 2019: RStudio v1.2.5033
  - [ ] `rstudio`
- [ ] December 16, 2019: `mcfly` v0.3.6 [VI keybindings](https://github.com/cantino/mcfly/releases/tag/v0.3.6)
- [ ] December 20, 2019: TigerVNC v1.10.1 [TigerVNC 1.10.1](https://github.com/TigerVNC/tigervnc/releases/tag/v1.10.1)
  - [ ] `tigervnc`
- [ ] December 21, 2019: Hexchat v2.14.3 [Changelog](https://hexchat.readthedocs.io/en/latest/changelog.html)
  - [ ] `hexchat`
- [ ] December 22, 2019: Stellarium v0.19.3 [Stellarium v0.19.3 has been released!](https://stellarium.org/release/2019/12/22/stellarium-0.19.3.html)
  - [ ] `stellarium`
- [ ] December 30, 2019: `ncompress` v4.2.4.6 [Various accumulated cleanups](https://github.com/vapier/ncompress/releases/tag/v4.2.4.6)
- [ ] December 30, 2019: `pngquant` v2.12.6
- [ ] December 31, 2019: PPP v2.4.8
  - [ ] `ppp`
- [ ] December 31, 2019: `ranger` v1.9.3
- [ ] January 5, 2020: `socat` v1.7.3.4 [CHANGES](http://www.dest-unreach.org/socat/doc/CHANGES)
- [ ] January 7, 2020: ZXing (C++) v1.0.8 [v1.0.8: Merge pull request #115 from marxin/add-missing-includes](https://github.com/nu-book/zxing-cpp/releases/tag/v1.0.8)
  - [ ] `zxing-cpp`
- [ ] January 9, 2020: uGet v2.2.3
  - [ ] `uget`
- [ ] January 16, 2020: `opam` v2.0.6 [2.0.6](https://github.com/ocaml/opam/releases/tag/2.0.6)
- [ ] January 19, 2020: KeePassXC v2.5.3 [Release 2.5.3](https://github.com/keepassxreboot/keepassxc/releases/tag/2.5.3)
  - [ ] `keepassxc`
- [ ] January 19, 2020: Midnight Commander v4.8.24
  - [ ] `mc`
- [ ] January 24, 2020: Blueman v2.1.2 [2.1.2](https://github.com/blueman-project/blueman/releases/tag/2.1.2)
  - [ ] `blueman`
- [ ] January 24, 2020: OUnit v2.2.2 [v2.2.2](https://github.com/gildor478/ounit/releases/tag/v2.2.2)
- [ ] January 25, 2020: PCI Utilities v3.6.4
  - [ ] `pciutils`
- [ ] January 27, 2020: `dhcpcd` v8.1.6
- [ ] January 27, 2020: Maxima v5.43.2
  - [ ] `maxima`
- [ ] January 29, 2020: QuiteRSS v0.19.3 [QuiteRSS 0.19.3 (29.01.2020)](https://quiterss.org/en/article/1523)
  - [ ] `quiterss`
- [ ] January 29, 2020: Yubico Authenticator for Desktop v5.0.2 [Yubico Authenticator for Desktop 5.0.2](https://github.com/Yubico/yubioath-desktop/releases/tag/yubioath-desktop-5.0.2)
  - [ ] `yubioath-desktop`
- [ ] January 29, 2020: Yubikey Manager v3.1.1 [yubikey-manager 3.1.1](https://github.com/Yubico/yubikey-manager/releases/tag/yubikey-manager-3.1.1)
  - [ ] `yubikey-manager`
- [ ] January 31, 2020: `lighttpd` v1.4.55
- [ ] January 31, 2020: `util-linux` v2.33.2
- [ ] February 1, 2020: Cgl v0.60.3
  - [ ] `cgl`
- [ ] February 1, 2020: CoinUtils v2.11.4
  - [ ] `coinutils`
- [ ] February 1, 2020: Osi v0.108.6
  - [ ] `osi`
- [ ] February 2, 2020: `ruby-cairo` v1.16.5
- [ ] February 4, 2020: Clp v1.17.5
  - [ ] `clp`
- [ ] February 9, 2020: `par2cmdline` v0.8.1 [v0.8.1: par2cmdline 0.8.1](https://github.com/Parchive/par2cmdline/releases/tag/v0.8.1)
- [ ] February 9, 2020: `ruby-pkg-config` v1.4.1
- [ ] February 10, 2020: NCurses Disk Usage v1.14.2
  - [ ] `ncdu`
- [ ] February 11, 2020: Ruby-Locale v2.1.3
  - [ ] `ruby-locale`
- [ ] February 13, 2020: HAProxy v2.0.13 [\[ANNOUNCE\] haproxy-2.0.13](https://www.mail-archive.com/haproxy@formilux.org/msg36327.html)
  - [ ] `haproxy`
- [ ] February 15, 2020: Mutt v1.13.4 [The Mutt E-Mail Client](http://www.mutt.org/)
  - [ ] `mutt`
- [ ] February 19, 2020: ALSA v1.2.2 [Changes v1.2.1.2 v1.2.2](https://alsa-project.org/wiki/Changes_v1.2.1.2_v1.2.2)
  - [ ] `alsa-lib`
  - [ ] `alsa-plugins`
  - [ ] `alsa-utils`
  - [ ] `alsa-lib+32`
  - [ ] `alsa-plugins+32`
- [ ] February 19, 2020: CryptoMiniSat v5.6.8 [CryptoMiniSat 5.6.8](https://github.com/msoos/cryptominisat/releases/tag/5.6.8)
  - [ ] `cryptominisat2`
  - [ ] Transitionalise `cryptominisat2` and move files to `cryptominisat`.
- [ ] February 19, 2020: Siege v4.0.5 [Siege 4.0.5](https://www.joedog.org/2020/02/19/siege-4-0-5/)
  - [ ] `siege`
- [ ] February 23, 2020: BOINC v7.16.5
  - [ ] `boinc`
- [ ] February 23, 2020: Claws Mail v3.17.5 [Latest News](https://www.claws-mail.org/news.php)
- [ ] February 24, 2020: CoinMP v1.8.4
  - [ ] `coin-mp`
- [ ] February 27, 2020: SPICE v0.14.3 [Major Changes in 0.14.3](https://gitlab.freedesktop.org/spice/spice/-/releases)
  - [ ] `spice`
- [ ] February 27, 2020: Zotero v5.0.84
  - [ ] `zotero`
- [ ] February 29, 2020: R v3.6.3 [R 3.6.3 is released](https://stat.ethz.ch/pipermail/r-announce/2020/000650.html)
  - [ ] `r`
- [ ] March 1, 2020: OpenBLAS v0.3.9 [OpenBLAS 0.3.9 version](https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.9)
  - [ ] `openblas`
- [ ] March 2, 2020: FriBiDi v1.0.9 [FriBiDi Version 1.0.9](https://github.com/fribidi/fribidi/releases/tag/v1.0.9)
  - [ ] `fribidi`
  - [ ] `fribidi+32`
- [ ] February 4, 2020: Hardware Data v0.333
  - [ ] `hwdata`
- [ ] February 9, 2020: Gost v2.11.0 [v2.11.0](https://github.com/ginuerzh/gost/releases/tag/v2.11.0)
  - [ ] `gost`
- [ ] March 3, 2020: PARI/GP v2.11.3 [Changelog](https://pari.math.u-bordeaux.fr/pub/pari/unix/pari-2.11.3.changelog)
  - [ ] `pari`
- [ ] March 8, 2020: BorgBackup v1.1.11 [Version 1.1.11 (2020-03-08)](https://borgbackup.readthedocs.io/en/stable/changes.html#version-1-1-11-2020-03-08)
  - [ ] `borgbackup`
- [ ] March 8, 2020: `pcsc-tools` v1.5.6 [Changelog](http://ludovic.rousseau.free.fr/softwares/pcsc-tools/Changelog)
- [ ] March 8, 2020: `xfce4-statusnotifier-plugin` v0.2.2
- [ ] March 8, 2020: Xfce4 Power Manager v1.6.6
  - [ ] `xfce4-power-manager`
- [ ] March 9, 2020: Cbc v2.10.5
  - [ ] `cbc`
- [ ] February 1, 2020: KCPTUN v20200321
  - [ ] `kcptun`
- [ ] March 10, 2020: `xrdp` v0.9.13 [xrdp v0.9.13](https://github.com/neutrinolabs/xrdp/releases/tag/v0.9.13)
- [ ] March 14, 2020: Debootstrap v1.0.123
  - [ ] `debootstrap`
- [ ] March 14, 2020: ModemManager 1.12.8 [ANN: ModemManager 1.12.8](https://lists.freedesktop.org/archives/modemmanager-devel/2020-March/007729.html)
  - [ ] `modemmanager`
- [ ] March 15, 2020: Inxi v3.0.38+1
  - [ ] `inxi`
- [ ] March 15, 2020: TeXworks v0.6.4 [TeXworks 0.6.4](https://github.com/TeXworks/texworks/releases/tag/release-0.6.4)
  - [ ] `texworks`
- [ ] March 18, 2020: Metasploit v5.0.81
  - [ ] `metasploit`
- [ ] March 18, 2020: Music Player Daemon v0.21.21
  - [ ] `mpd`
- [ ] March 19, 2020: Wireguard Tools v1.0.20200319
  - [ ] `wireguard-tools`
- [ ] March 20, 2020: DeadBeef v1.8.3
  - [ ] `deadbeef`
- [ ] March 21, 2020: Greybird v3.22.12 [Greybird 3.22.12](https://github.com/shimmerproject/Greybird/releases/tag/v3.22.12)
  - [ ] `greybird`
- [ ] March 21, 2020: RetroArch v1.8.5 [RetroArch 1.8.5 released!](https://www.libretro.com/index.php/retroarch-1-8-5-released/)
  - [ ] `retroarch`
  - *Update assets.*
- [ ] March 22, 2020: Toxcore v0.2.11 [v0.2.11](https://github.com/TokTok/c-toxcore/releases/tag/v0.2.11)
  - [ ] `toxcore`
- [ ] March 23, 2020: Rambox v0.7.4 [0.7.4](https://github.com/ramboxapp/community-edition/releases/tag/0.7.4)
  - [ ] `rambox`
- [ ] March 24, 2020: V2Ray v4.23.1 [V4.23.1](https://github.com/v2ray/v2ray-core/releases/tag/v4.23.1)
  - [ ] `v2ray`
- [ ] March 29, 2020: X.Org Server v1.20.8 [\[ANNOUNCE\] xorg-server 1.20.8](https://lists.x.org/archives/xorg-announce/2020-March/003041.html)
  - [ ] `xorg-server`
- [x] June 10, 2020: Trojan v1.16.0 [Trojan 1.16.0](https://github.com/trojan-gfw/trojan/releases/tag/v1.16.0)
  - [x] `trojan`
- [x] June 11, 2020: DNSCrypt Proxy v2.0.44 [2.0.44](https://github.com/DNSCrypt/dnscrypt-proxy/releases/tag/2.0.44)
  - [x] `dnscrypt`
- [x] June 12, 2020: CA Certs v20200612 [Bug 1645186 - Add certSIGN Root CA G2 root cert r=KathleenWilson](https://github.com/nss-dev/nss/commit/44638db4eed579df3622a6378c4889b7d036fd96#diff-3f515d5009adff2f9ba14a0d62a302ce)
  - [x] `ca-certs`
- [x] June 16, 2020: Intel Microcode v20200616 [microcode-20200616 release](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20200616)
  - [x] `intel-ucode`
- [x] June 17, 2020: Min Browser v1.15.0 [1.15: Improved scripting support](https://github.com/minbrowser/min/releases/tag/v1.15.0)
  - [x] `min`
- [x] June 23, 2020: Brave Browser v1.10.97 [Desktop Release Channel v1.10.97](https://github.com/brave/brave-browser/releases/tag/v1.10.97)
  - [x] `brave-browser`
- [x] June 24, 2020: Telegram Desktop v2.1.13 [v 2.1.13](https://github.com/telegramdesktop/tdesktop/releases/tag/v2.1.13)
  - [x] `telegram-desktop`
- [x] June 30, 2020: Linux Kernel v5.7.7 [ChangeLog](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.7.7)
  - [x] `linux-kernel`
  - [x] `linux+kernel`
  - [x] `kernel-tools`
- [x] June 30, 2020: Linux Kernel (LTS) v5.4.50 [ChangeLog](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.50)
  - [x] `linux-kernel-lts`
  - [x] `linux+kernel+lts`
- [x] July 1, 2020: KCPTUN v20200701 [Improved sliding window](https://github.com/xtaci/kcptun/releases/tag/v20200701)
  - [x] `kcptun`
- [x] July 2, 2020: Hardware Database v0.337
  - [x] `hwdata`
- [x] July 2, 2020: Linux Firmware v20200702 (Git snapshot) [Merge https://github.com/bgodavar/qca_wcn3991 into main](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=74ac3b5a7e7962cd1135ef691c72d48cb5210251)
  - [x] `firmware-free`
  - [x] `firmware-nonfree`
- [x] July 2, 2020: Opera v69.0.3686.49
  - [x] `opera`
- [x] July 7, 2020; SeaMonkey v2.53.3 [SeaMonkey 2.53.3 released](https://www.seamonkey-project.org/news)
  - [x] `seamonkey`
- [x] July 7, 2020: groups/plasma 5.19.3 [5.19.3](https://kde.org/announcements/plasma-5.19.3)
- [x] July 10, 2020: intel-media-driver 20.2.0 [20.2.0](https://github.com/intel/media-driver/releases/tag/intel-media-20.2.0)
- [x] July 10, 2020: nextcloud-client 2.6.5 [2.6.5](https://github.com/nextcloud/desktop/releases/tag/v2.6.5)

### Testing-Proposed => Testing
- [x] Jan 20, 2019: KeePassX v2.0.3+20190120 [Fix build with Qt 5.11](https://github.com/keepassx/keepassx/commit/1682ab90243ef93449a981eb7b8046b629720488)
  - [x] `keepassx`
- [x] March 4, 2019: ProxyMan v3.1.2 [Bug fixes in v3](https://github.com/himanshub16/ProxyMan/releases/tag/v3.1.2)
  - [x] `proxyman`
- [ ] August 5, 2019: Volk v2.0.0 [Release 2.0.0](https://github.com/gnuradio/volk/releases/tag/v2.0.0)
  - [ ] `volk`
- [ ] August 10, 2019: GNU Radio v3.8.0.0
  - [ ] `gnuradio`
- [ ] August 14, 2019: Boost v1.71.0 [Version 1.71.0](https://www.boost.org/users/history/version_1_71_0.html)
  - [ ] `boost`
  - [ ] _Rebuild [reverse dependencies](https://packages.aosc.io/revdev/boost)._
- [ ] August 19, 2019: ASIO v1.14.0
  - [ ] `asio`
  - _Defer if breaks other packages._
- [ ] August 20, 2019: GNU Scientific Library v2.6 [GNU Scientific Library 2.6 Released](http://lists.gnu.org/archive/html/info-gnu/2019-08/msg00004.html)
  - [ ] `gsl`
- [ ] September 3, 2019: Ceph v14.2.3
  - [ ] `ceph`
  - *Drop?*
- [ ] September 5, 2019: `libsigc++` v3.0.0 [3.0.0](https://github.com/libsigcplusplus/libsigcplusplus/releases/tag/3.0.0)
- [ ] September 19, 2019: OpenEXR v2.4.0 [v2.4.0](https://github.com/openexr/openexr/releases/tag/v2.4.0)
  - [ ] `openexr`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/openexr)._
- [ ] September 22, 2019: Nimble v0.11.0 [v0.11.0](https://github.com/nim-lang/nimble/releases/tag/v0.11.0)
  - [ ] `nimble`
- [ ] October 3, 2019: ICU v65 [ICU 65](http://site.icu-project.org/download/65)
  - [ ] `icu`
  - [ ] _Rebuild [reverse-dependencies](https://packages.aosc.io/revdep/icu)._
- [ ] October 7, 2019: Open MPI v4.0.2 [NEWS](https://raw.githubusercontent.com/open-mpi/ompi/v4.0.x/NEWS)
  - [ ] `openmpi`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/openmpi)._
- [ ] October 10, 2019: GEOS v3.8.0
  - [ ] `geos`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/geos)._
- [ ] October 11, 2019: Racer v2.1.28 [racer version 2.1.28](https://github.com/racer-rust/racer/releases/tag/v2.1.28)
  - [ ] `racer`
- [ ] October 20, 2019: PostGIS v3.0.0
  - [ ] `postgis`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/postgis)._
- [ ] October 24, 2019: VirtualGL v2.6.3 [2.6.3](https://github.com/VirtualGL/virtualgl/releases/tag/2.6.3)
  - [ ] `virtualgl`
  - [ ] `virtualgl+32`
- [ ] October 27, 2019: VeraCrypt v1.24+1 (v1.24-Hotfix1)
  - [ ] `veracrypt`
- [ ] October 29, 2019: Protocol Buffers v3.10.1 [Protocol Buffers v3.10.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.10.1)
  - [ ] `protobuf`
  - _Rebuild [reverse-dependencies](https://packages.aosc.io/revdep/protobuf). Defer if build errors are unresolvable._
- [ ] November 8, 2019: CGAL v5.0 [CGAL-5.0](https://github.com/CGAL/cgal/releases/tag/releases%2FCGAL-5.0)
  - [ ] `cgal`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/cgal). Defer to v4.14.1 if build errors are unresolvable._
- [ ] November 9, 2019: GRASS v7.8.1 [GRASS GIS 7.8.1](https://trac.osgeo.org/grass/wiki/Release/7.8.1-News)
  - [ ] `grass`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/grass)._
- [ ] November 12, 2019: Closure Compiler v20191111
  - [ ] `closure-compiler`
- [ ] November 26, 2019: Julia v1.3.0 [v1.3.0](https://github.com/JuliaLang/julia/releases/tag/v1.3.0)
  - [ ] `julia`
- [ ] November 27, 2019: Poppler v0.83.0 [Poppler](https://poppler.freedesktop.org/)
  - [ ] `poppler`
  - [ ] `poppler+32`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/poppler). Defer if build errors are unresolvable._
- [ ] November 30, 2019: Coq v8.10.2 [Coq 8.10.2](https://github.com/coq/coq/releases/tag/V8.10.2)
  - [ ] `coq`
- [ ] December 6, 2019: QGIS v3.10.1
  - [ ] `qgis`
- [ ] December 10, 2019: `mediastreamer` v4.3.0
- [ ] December 10, 2019: `ortp` v4.3.0
- [ ] December 16, 2019: RethinkDB v2.4.0
  - [ ] `rethinkdb`
- [ ] February 26, 2020: NCBI VDB v2.10.4
  - [ ] `ncbi-vdb`
- [x] April 19, 2020: bleachbit v4.0.0
  - [x] `bleachbit`
- [x] June 17, 2020: eclipse v4.16 [Eclipse 4.16 - New and Noteworthy](https://www.eclipse.org/eclipse/news/4.16/)
  - [x] `eclipse`
- [ ] June 19, 2020: Free Pascal v3.2.0 Also added aarch64 support
  - [x] `fpc`
  - [ ] `lazarus`
- [x] June 30, 2020: Exiv2 v0.27.3 [Changelog](https://www.exiv2.org/changelog.html#v0.27.3)
  - [x] `exiv2`
  - [x] _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/exiv2)._
- [ ] July 3, 2020: dolphin-emu 20200703
  - [ ] `dolphin-emu`
- [ ] July 4, 2020: wine 5.12
- [x] July 7, 2020: Ardour 6.2 [Release 6.2](https://github.com/Ardour/ardour/releases/tag/6.2)
  - [x] `ardour`
- [ ] August 11, 2020: `qemu` 5.1.0 [Changelog/5.1](https://wiki.qemu.org/ChangeLog/5.1)
- [ ] Sept 6, 2020: `digikam` 7.1.0 [digiKam 7.1.0 is released](https://www.digikam.org/news/2020-09-06-7.1.0_release_announcement/)
  - [ ] `digikam`

### Deferred to Core 8 (Summer 2020?)

- February 1, 2020: Glibc v2.31 [Release/2.31](https://sourceware.org/glibc/wiki/Release/2.31)
  - `glibc`

## Packages to Add

- [ ] `monero` v0.15.0.5
- [ ] `turtlecoin` v0.24.0
- [ ] `netbsd-tools` v9.0
- [ ] `time` v1.9
- [ ] `rt-tests` v1.8
- [ ] `free42` v2.5.18
- [x] `cli-visualizer` v1.8
- [x] `libldac` v2.0.2.3
- [x] `pulseaudio-modules-bt` v1.4
- [x] `libtomcrypt` v1.18.2
- [ ] `stepmania` v5.1.0+20200705
- [x] `firebird-emu` v1.5-1
- [x] `fcitx5-material-color` 20200410
- [x] `oolite` 1.90
- [x] `nvidia-prime` v1.0
- [x] `fsearch` v0.1~beta4
- [x] `compsize` v1.3
- [x] `libsignal-protocol-c` v2.3.3-1
- [x] `dino` v0.1.0
- [x] `hstspreload` 20200717
- [x] `httpcore` v0.9.1
- [x] `rfc3986` v1.4.0
- [x] `sniffio` v1.1.0
- [x] `httpx` v0.13.3
- [ ] `bcc` v0.15.0
- [ ] `procmon` v1.0
- [ ] `openrgb` v0.3
- [x] `srain` v1.1.1
- [ ] `frp` v0.33.0
- [x] `yarl` v1.5.0
- [x] `async-timeout` v3.0.1
- [x] `multidict` v4.7.6
- [x] `aiohttp` v3.6.2
- [x] `autojump-rs` v0.4.0+22.5.3
- [x] `boinctui` v2.5.0

## Packages to Drop

TBD

## OS-wide Projects

- [ ] Meta: Apply HTTPS to all packages where available (including `ftp://` sources).

## Help Wanted

TBD

## Documentation Work

- [Security updates for 2019](https://aosc.wiki/aosa/archive/2019)
- [Security updates for 2020](https://aosc.wiki/aosa/archive/2020)

## See Also

- [Open Security Issues](https://github.com/AOSC-Dev/aosc-os-abbs/issues?q=is%3Aissue+is%3Aopen+label%3Asecurity)
- [Repology](https://repology.org/repository/aosc)

## Deferred Items

- March 24, 2020: LLVM v10.0.0 [\[llvm-announce\] LLVM 10.0.0 Release](http://lists.llvm.org/pipermail/llvm-announce/2020-March/000087.html)

**Comment by MingcongBai** [comment]
Summer 2020 is now live, all dates pushed back by 7 days due to delay from the last cycle.

**Comment by MingcongBai** [comment]
Delaying cycle endpoint by three weeks, new schedule as follows:

- September 29, 2020: Freeze Testing-Proposed.
  - Merge all changes to Testing for updates.
  - Transfer all skip-ahead changes to Experimental.
  - All non-complete entries on Testing-Proposed => Testing to be deferred to Fall 2020.
- September 29, 2020: Commence draft on Fall 2020 Iteration Plan.
- October 13, 2020: Merge all changes to Stable, iteration completion.

**Comment by MingcongBai** [comment]
Iteration frozen, this will be our last until topic-base iterations starts rolling.
